dlib/CMakeLists.txt

```diff
@@ -422,7 +423,7 @@ if (NOT TARGET dlib)
          find_package(GIF QUIET)
          if (GIF_FOUND)
             set (dlib_needed_includes ${dlib_needed_includes} ${GIF_INCLUDE_DIR})
-            set (dlib_needed_libraries ${dlib_needed_libraries} ${GIF_LIBRARY})
```

Comment:
This feeds into more than just pkgconfig.  In particular, into `target_link_libraires`.  So this can't just be elided.